src/components/Layout/Layout.module.scss

```diff
@@ -1,6 +1,9 @@
 .layout {
   margin: 0 auto;
   max-width: 100%;
+  flex-direction: column;
+    display: flex;
+    min-height: 100%;
```

Comment:
Can we please have the lines properly indented?

Reply:
@swarajpure its fixed

Reply:
![image](https://user-images.githubusercontent.com/24789070/105042421-3e2c2800-5a8a-11eb-8cb2-1b3242525fba.png)